crates/bevy_render/src/primitives/mod.rs

```diff
@@ -109,23 +151,46 @@ impl HalfSpace {
         Vec3A::from(self.normal_d)
     }
 
-    /// Returns the distance from the origin to the bisecting plane along the plane's unit normal vector.
-    /// This distance helps determine the position of a point `p` on the bisecting plane, as per the equation `n.p + d = 0`.
+    /// Returns the signed distance from the bisecting plane to the origin along
+    /// the plane's unit normal vector.
     #[inline]
     pub fn d(&self) -> f32 {
         self.normal_d.w
     }
 
-    /// Returns the bisecting plane's unit normal vector and the distance from the origin to the plane.
+    /// Returns the bisecting plane's unit normal vector and the signed distance
+    /// from the plane to the origin.
     #[inline]
     pub fn normal_d(&self) -> Vec4 {
         self.normal_d
     }
 }
 
-/// A frustum made up of the 6 defining half spaces.
-/// Half spaces are ordered left, right, top, bottom, near, far.
-/// The normal vectors of the half spaces point towards the interior of the frustum.
+/// A frustum defined as the intersection of 6 [`HalfSpace`].
+///
+/// Usually an apex-truncated square pyramid (a pyramid without the top) or a cuboid.
+///
+/// Half spaces are ordered left, right, top, bottom, near, far. The normal vectors
+/// of the half-spaces point towards the interior of the frustum.
+///
+/// A frustum component is used on an entity with a [`Camera`] component
+/// to determine which entities will be considered for rendering by this camera.
+/// All entities with an [`Aabb`] component that does not intersect with the frustum
```

Comment:
```suggestion
/// All entities with an [`Aabb`] component that are not contained by (or overlapping the boundary of) the frustum
```

Reply:
I changed "overlapping" to "crossing" since technically the boundary is not part of the frustum and if the Aabb just touches the boundary they overlap but the entity will not be considered.